### ISSUE TITLE
Presto scramble should keep original partitions

### ISSUE THREAD

**Issue by commercial-hippie** [mozafari/verdictdb#303]
As the title suggests, I think the scrambles should keep the original partitions and just prepend the `verdictdbblock` partition.

Currently it doesn't work that way.. But looking at the code I think that's how it was intended to work?

**Comment by commercial-hippie** [comment]
I just ran some more tests one this, and while I think it is a good idea it looks like it might cause more errors for tables that have multiple partitions already.

https://groups.google.com/forum/#!topic/presto-users/5gFbvUoOF5I

**Comment by pyongjoo** [comment]
@dongyoungy Can you check if the partition count limit is (1) per table, or (2) per insertion or create statement? If the latter is the case, we can still easily support daily data ingestion. (under the assumption that the insertion date is the only column on which a table is partitioned).

**Comment by pyongjoo** [comment]
I believe this issue is related to https://github.com/mozafari/verdictdb/issues/302. So, let's see what @commercial-hippie says about his requirements. If he indeed chooses to pursue daily data ingestion, we can handle two issues (https://github.com/mozafari/verdictdb/issues/302 and this) together.

**Comment by commercial-hippie** [comment]
@pyongjoo I think having the partitions would be used in both cases.. When we add new data as well as when we backfill older data..

I do have a modified version of Verdict that I used to test this but I ran into the hive error mentioned in comment 2.

We do have quite a few partitions currently which have helped speed up non-verdict queries a lot.  I might look into only adding one of the most used partitions to the verdict scrambles and increasing "hive.max-partitions-per-writers", instead of having all the partitions.

I'll do this testing using my modified version of Verdict and report back here on the results.
